File: apollo-runtime/src/main/java/com/apollographql/apollo/internal/subscription/ApolloSubscriptionException.java
package com.apollographql.apollo.internal.subscription;

import com.apollographql.apollo.exception.ApolloException;

@SuppressWarnings("WeakerAccess")
public class ApolloSubscriptionException extends ApolloException {

  public ApolloSubscriptionException(String message) {
    super(message);
  }

  public ApolloSubscriptionException(String message, Throwable cause) {
    super(message, cause);
  }
}

File: apollo-runtime/src/main/java/com/apollographql/apollo/internal/subscription/ApolloSubscriptionException.kt
package com.apollographql.apollo.internal.subscription

import com.apollographql.apollo.exception.ApolloException

open class ApolloSubscriptionException : ApolloException {
  constructor(message: String?) : super(message!!)
  constructor(message: String?, cause: Throwable?) : super(message!!, cause!!)
}